src/views/index.js

```diff
@@ -1,6 +1,7 @@
-// IIFE - immediately invoked function expression
+// IIFE - immediately invoked function expression..
+const interview = {};
 window.onload = function () {
     const appEl = document.querySelector('#app');
     const isLogged = !!sessionStorage.userLogged;
-    isLogged ? interview.modules.evaluations.init() : interview.modules.login.init();
-}()
\ No newline at end of file
+    isLogged ? interview.loadModule(interview.modules.evaluations) : interview.loadModule(interview.modules.login);
```

Comment:
`interview.loadModule(isLogged ? interview.modules.evaluations : interview.modules.login);` for a more concise version, since we'd be loading a module whether we're logged in or not.